### ISSUE TITLE
Is the v15.x branch still useful in late 2023

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/postgresql-feedstock#190]
### Comment:

Seems like the migration was closed 1 year ago. Are there still some known packages that use postgresql 15
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3902

I found: https://github.com/conda-forge/pgvector-feedstock/pull/19

**Comment by akrherz** [comment]
I think it is useful and I personally use it as my very large conda environment is always in dependency hell, so I am still stuck with postgresql 15.  Having said that, I think I am presently only stuck because of the very ancient `ncl` not being able to move forward with libjpeg-turbo

**Comment by bollwyvl** [comment]
Yep. Still useful in situations where the `postgresql` on the other end of the pipe can't be controlled for whatever reason.
